### PR TITLE
updated styling on document records table

### PR DESCRIPTION
**Pull request by flutistar**
#22806

- [x] Document Record results showing incorrect results
- [x] Padding should be 20px above the sort by column
- [x] Missing drop-downs in header: Entity Type and Columns to Show
- [x] Breaklines should span the whole width of the table
- [x] Sort and Filter By text should be 14px #495057
- [x] field heigh should be 44px 
- [x] missing dropdown button in Document Type field
- [x] Missing icon in Date Range Picker 
- [x] Missing 'Description' column (see Sort By / Filter By Styling in figma File)
- [x] Corder Radius's in fields should be 4px 
- [x] Date range picker is showing single date selector. (See Sort By / Filter By Styling and Varients - Date Range Picker) *Again this may be for ticket #23120
- [x] Tool Tip width should be 265px
- [x] Icon colour should be #1669BB
- [x] Buttons | Corder Radius's should be 4px 

image

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://documents-ui-dev--pr-65-u7og31dn.web.app

File: document-service/documents-ui/src/components/InputMultiSelector/index.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    </template>
+    <template #option="optionProps">
+      <UCheckbox
+        v-model="optionProps.selected"
+        :name="optionProps.option[valueAttribute]"
+        :label="optionProps.option[optionAttribute]"
+      />
+    </template>
+  </USelectMenu>
+</template>
```
Comment (cameron-eyds) [inline document-service/documents-ui/src/components/InputMultiSelector/index.vue:62]: Very cool, will have a look at this

Comment (flutistar) [reply document-service/documents-ui/src/components/InputMultiSelector/index.vue:62]: Thanks

File: document-service/documents-ui/src/utils/documentTypes.ts [hunk 2]
```diff
@@ -1,13 +1,11 @@
-import type { DocumentClassIF } from '~/interfaces/document-types-interface'
-
 /**
  * @file documentTypes.ts
  * @description Exports an object categorizing document types by entity type. Each category includes:
  * - `class`: Entity class.
  * - `prefixes`: Array of document prefixes.
  * - `documents`: Array of document details (type, description, product code).
  */
-export const documentTypes: Array<DocumentClassIF> = [
```
Comment (cameron-eyds) [inline document-service/documents-ui/src/utils/documentTypes.ts:10]: Why the removal of the interface?

**Review by cameron-eyds — COMMENTED** [review]
Looks pretty good!

Just a minor question.

**Review by doug-lovett — APPROVED** [review]
Looks great, lots of work here.

File: document-service/documents-ui/src/utils/documentTypes.ts (hunk 2 above)
Comment (flutistar) [reply document-service/documents-ui/src/utils/documentTypes.ts]: My mistake, reverted!

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://documents-ui-dev--pr-65-u7og31dn.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://documents-ui-dev--pr-65-u7og31dn.web.app

**Review by cameron-eyds — APPROVED** [review]
Looks great, thanks! 

I think you should go first and i'll deal with the conflicts :D

**Pull request merged**